q2_vsearch/tests/test_merge_pairs.py

```diff
@@ -87,11 +114,47 @@ def test_merge_pairs(self):
                 len(seq_lengths),
                 default_exp_sequence_counts[str(fastq_name)])
 
+    def test_unmerged_pairs(self):
+        with redirected_stdio(stderr=os.devnull):
+            _, unmerged = merge_pairs(self.input_seqs)
+
+        # manifest is as expected
+        self._test_manifest_unmerged(unmerged)
+
+        # expected number of fastq files are created
+        output_fastqs = list(unmerged.sequences.iter_views(FastqGzFormat))
+        self.assertEqual(len(output_fastqs), 6)
+
+        default_exp_sequence_counts = {
+            'BAQ2687.1_0_L001_R1_001.fastq.gz': 31,
```

Comment:
I'm guessing these numbers came from looking subtracted the expected merged sequence counts from the input sequence counts? Sounds good if so - just confirming.